.gitignore

```diff
@@ -1,6 +1,10 @@
 .DS_Store
 .vs
 
+debug/
+target/
```

Comment:
This won't be excluded already with **/target?

Reply:
Ah, thanks!